apps/ffdweb-site/src/app/_constants/icons.ts

```diff
@@ -0,0 +1,10 @@
+import { CaretRight } from '@phosphor-icons/react/dist/ssr'
+
+export const ICON_CONFIG = {
```

Comment:
I don't think it's this general - More like `CARD_CTA_ICON_CONFIG`? Which doesn't need to be nested at the moment either.